diff --git a/typings.d.ts b/typings.d.ts
index c83b11b6a2735bc763cddba36c20213c90a60cbf..1b113f1cd3ef313bdb9ce0ba38e5bd27fc5ada22 100644
--- a/typings.d.ts
+++ b/typings.d.ts
@@ -368,9 +368,9 @@ type MaybeGenerated<TValue, TOptions, TProcessOnCreate extends boolean> = TOptio
 } ? TValue | null : TOptions extends {
     autoincrement: true;
 } ? Generated<TValue> : TOptions extends {
-    default: true;
+    default: unknown;
 } ? Generated<TValue> : TOptions extends {
-    defaultRaw: true;
+    defaultRaw: unknown;
 } ? Generated<TValue> : TProcessOnCreate extends false ? TValue : TOptions extends {
     onCreate: Function;
 } ? Generated<TValue> : TValue;
