diff --git a/lib/es/vite.mjs b/lib/es/vite.mjs
index 0da6294e82f05d5a858f8b3740ecdfd8b2dd239c..50572f47992f83f7c3e99030649ab68c44e171e5 100644
--- a/lib/es/vite.mjs
+++ b/lib/es/vite.mjs
@@ -68,6 +68,7 @@ function attachViteHooks(plugin) {
             } catch {}
           }
         }, 150);
+        interval.unref?.();
         server.httpServer?.once('close', () => clearInterval(interval));
       }
     },
diff --git a/lib/vite.js b/lib/vite.js
index bf3df4c8b81574dafae3f626767706455f03854d..2751c00d8c8d00680065712c67d9544e7ffcc3cf 100644
--- a/lib/vite.js
+++ b/lib/vite.js
@@ -75,6 +75,7 @@ function attachViteHooks(plugin) {
             } catch {}
           }
         }, 150);
+        interval.unref?.();
         server.httpServer?.once('close', () => clearInterval(interval));
       }
     },
